ui/apps/platform/src/types/search.ts

```diff
@@ -33,3 +33,10 @@ export type GraphQLSortOption = {
     id: string;
     desc: boolean;
 };
+
+export type SearchQueryOptions = {
```

Comment:
Just for your awareness, an open pull request has a new type that might have related purpose:
https://github.com/stackrox/stackrox/pull/11295/files#diff-225d1d3f34456ffd1dd6dfcd00d49805cf0a1e21ccbeb524df3fea38b2802753R3-R8